eo-maven-plugin/src/test/java/org/eolang/maven/ProbeMojoTest.java

```diff
@@ -122,6 +162,134 @@ void findsProbesInOyRemote(@TempDir final Path temp) throws IOException {
         );
     }
 
+    @Test
+    @ExtendWith(OnlineCondition.class)
+    @Disabled
+    void findsProbesWithVersionsInOneObjectionary(@TempDir final Path temp) throws IOException {
+        final CommitHash hash = new CommitHashesMap.Fake().get("0.28.10");
+        final String object = "org.eolang.io.stdout|9b88393";
+        final FakeMaven maven = new FakeMaven(temp)
+            .with("objectionaries", new OjsDefault().with(hash, new OyRemote(hash)))
+            .with("withVersions", true)
+            .withProgram(
+                "+package org.eolang.custom\n",
+                "[] > main",
+                "  QQ.io.stdout|0.28.10 > @",
+                "    \"Hello world\""
+            )
+            .execute(new FakeMaven.Probe());
+        MatcherAssert.assertThat(
+            String.format(
+                "Tojos should have contained versioned object %s after probing, but they didn't",
+                object
+            ),
+            maven.externalTojos().contains(object),
+            Matchers.is(true)
+        );
+        MatcherAssert.assertThat(
+            "First entry of tojos after probing should have contained one probed object, but it didn't",
+            ProbeMojoTest.firstEntry(
```

Comment:
@maxonfjvipon Looks like you are tying to check internals instead of checking actual results (some sort of encapsulation violation). Or I'm wrong here? It's a lot of questions actually:
- why it should be the "first" entity?
- why "1"?
And so on

Reply:
@volodya-lombrozo 
1. I believe, encapsulation violation is something about `get`, `set` and reflection. Here we just read the file and get first row from it.
2. It should be "first" because only one expect this first tojo was added on `register` step and contained `probed` field after `probe` step
3. "1" because we expect that `probed` field would contain value "1"
4. This test class is full of using this static method and it seems convenient

Reply:
@maxonfjvipon The "first entity of the foreign tojos" looks strange. It looks like we are tying to scan an intermediate state which we are trying to read in the middle of the process. I'm afraid it could lead to a fragile implementation. Could we check the result somehow else? (Or to skip that check at all?). 

If not, can we add a puzzle which will request the refactoring of that test class in order to make it more readable?